releases/release-5.2.4.md

```diff
@@ -0,0 +1,193 @@
+        - (dup: release-5.3.1.md > Bug fixes> Tools> TiCDC)- Fix the issue that `stopped` changefeeds resume automatically after a cluster upgrade [#3473](https://github.com/pingcap/tiflow/issues/3473)
+        - (dup: release-5.3.1.md > Bug fixes> Tools> TiCDC)- Fix the issue of overly frequent warnings caused by MySQL sink deadlock [#2706](https://github.com/pingcap/tiflow/issues/2706)
+        - (dup: release-5.3.1.md > Bug fixes> Tools> TiCDC)- Fix the bug that the `enable-old-value` configuration item is not automatically set to `true` on Canal and Maxwell protocols [#3676](https://github.com/pingcap/tiflow/issues/3676)
+        - (dup: release-5.3.1.md > Bug fixes> Tools> TiCDC)- Fix the issue that Avro sink does not support parsing JSON type columns [#3624](https://github.com/pingcap/tiflow/issues/3624)
+        - (dup: release-5.3.1.md > Bug fixes> Tools> TiCDC)- Fix the negative value error in the changefeed checkpoint lag [#3010](https://github.com/pingcap/tiflow/issues/3010)
+        - (dup: release-5.4.0.md > Bug fixes> Tools> TiCDC)- Fix the OOM issue in the container environment [#1798](https://github.com/pingcap/tiflow/issues/1798)
+        - (dup: release-4.0.16.md > Bug fixes> Tools> TiCDC)- Fix the memory leak issue after processing DDLs [#3174](https://github.com/pingcap/tiflow/issues/3174)
+        - Fix chengefeed getting stuck when tables are repeatedly scheduled in the same node [#4464](https://github.com/pingcap/tiflow/issues/4464)
+        - Fix a bug that openapi may be stuck when pd is abnormal [#4778](https://github.com/pingcap/tiflow/issues/4778)
+        - Fix stale metrics caused by owner changes. [#4774](https://github.com/pingcap/tiflow/issues/4774)
+        - Fix stability problem in workerpool, which is used by Unified Sorter. [#4447](https://github.com/pingcap/tiflow/issues/4447)
+        - Fix kv client cached region metric could be negative. [#4300](https://github.com/pingcap/tiflow/issues/4300)
+
+    + TiDB Lightning
+
+        - (dup: release-5.4.0.md > Bug fixes> Tools> TiDB Lightning)- Fix the issue of wrong import result that occurs when TiDB Lightning does not have the privilege to access the `mysql.tidb` table [#31088](https://github.com/pingcap/tidb/issues/31088)
+        - (dup: release-6.0.0-dmr.md > Bug fixes> Tools> TiDB Lightning)- Fix the checksum error “GC life time is shorter than transaction duration” [#32733](https://github.com/pingcap/tidb/issues/32733)
+        - (dup: release-6.0.0-dmr.md > Bug fixes> Tools> TiDB Lightning)- Fix a bug that TiDB Lightning may not delete the metadata schema when some import tasks do not contain source files [#28144](https://github.com/pingcap/tidb/issues/28144)
+        - (dup: release-5.1.4.md > Bug fixes> Tools> TiDB Lightning)- Fix the issue that TiDB Lightning does not report errors when the S3 storage path does not exist [#28031](https://github.com/pingcap/tidb/issues/28031) [#30709](https://github.com/pingcap/tidb/issues/30709)
```

Comment:
```suggestion
        - (dup: release-5.1.4.md > Bug fixes> Tools> TiDB Lightning)- Fix the issue that TiDB Lightning does not report errors when the S3 storage path does not exist [#28031](https://github.com/pingcap/tidb/issues/28031) [#30709](https://github.com/pingcap/tidb/issues/30709)
```suggestion
        -  Fix error occurs if iterate exceeds 1000 keys of GCS  [#30377](https://github.com/pingcap/tidb/issues/30377) 
```